x-pack/libbeat/management/input_reload_test.go

```diff
@@ -0,0 +1,213 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package management
+
+import (
+	"fmt"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/joeshaw/multierror"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+
+	"github.com/elastic/beats/v7/libbeat/common"
+	"github.com/elastic/beats/v7/libbeat/common/reload"
+	"github.com/elastic/elastic-agent-client/v7/pkg/client"
+	"github.com/elastic/elastic-agent-client/v7/pkg/proto"
+)
+
+func TestInputReload(t *testing.T) {
```

Comment:
This test takes 10s to run? Can we speed it up? I assume we are just using the default reload timeout.

```
 gotest ./x-pack/libbeat/management/... -run TestInputReload -v
=== RUN   TestInputReload
--- PASS: TestInputReload (11.10s)
PASS
ok      github.com/elastic/beats/v7/x-pack/libbeat/management   11.309s
```